recipes/awslambdaric/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "awslambdaric" %}
+{% set version = "2.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/awslambdaric-{{ version }}.tar.gz
+  sha256: dad646f566aa7ec9b7179f16ca6741a2bea148abec6ed5947f86d00607e0a9a2
```

Due to an update in bintutils we need to use a more recent version of backwards-cpp library. 
Please see awslabs/aws-lambda-cpp#94 for more details

```suggestion
  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/awslambdaric-{{ version }}.tar.gz
    sha256: dad646f566aa7ec9b7179f16ca6741a2bea148abec6ed5947f86d00607e0a9a2
    patches:
      # Replace the pinned simplejson constraint with >=3.17.2,<3.18
      - relax-simplejson.patch
      # Please see awslabs/aws-lambda-cpp#94, remove once aws-lambda-cpp is upgraded to 0.2.7
      - aws-lambda-cpp-0.2.6-update-backward-cpp.patch
  - url: https://raw.githubusercontent.com/bombela/backward-cpp/v1.6/backward.hpp
    sha256: 404aa26ecc36be08f92c03b05505561e064f6f1f7cffcbda77a7d4959e4c07f2
    folder: deps/patches
    fn: backward.h
```